aiidalab_widgets_base/codes.py

```diff
@@ -19,87 +20,116 @@ def valid_aiidacode_args(arguments):
 
 
 class CodeDropdown(ipw.VBox):
-    """Code selection widget."""
+    """Code selection widget.
+    Attributes:
+        selected_code(Unicode or Code): Trait that points to the selected Code instance.
+            It can be set either using Code or code's name. It is linked to the 'value' trait
```

Comment:
```suggestion
            It can be set either to a AiiDA Code instance or to a code label (will be automatically replaced by the corresponding Code instance).
```